docs/src/main/sphinx/installation/try-it.rst

```diff
@@ -0,0 +1,265 @@
+Try Trino with Docker
+======================
+
+This guide focuses on your first ten minutes
+using Trino, and relies on Docker to simplify the installation and
+ensure that the CLI runs on your platform of choice.
+
+This Docker container includes both the Trino server and the 
+Trino command line interface (CLI), which provides a terminal-based,
+interactive shell for running queries and inspecting catalog structures
+in large data clusters. In production, it's unlikely that your server 
+and client will reside in the same location. However, to keep this guide 
+short and sweet, we are going to run both from your machine.
+
+In this guide, you'll learn how to:
+
+-  Install and run the Trino Interactive CLI
+-  Call Trino from the terminal
+
+Prerequisites
+-------------
+
+Trino runs best on Linux. Our Docker container is built on 
```

Comment:
Hmmm. This page's title promises me a Quickstart for Trino *with Docker*, so I want to know what Docker runs best on. What Trino runs on _in the container_ is assumed to be ready to run, or it wouldn't be offered as a runnable container.  It's true but out of place to say that Trino runs best on Linux. The important point for this page is that you can get it running from any OS starting point. 

Rework this paragraph to convey:

- Our Docker container encapsulates Trino on Linux for 64-bit Intel platforms
- **For evaluation and testing only**, you can run the Trino container on macOS or Windows hosts
- Run the Trino container only on a Linux host for a production environment

Reply:
Should we call out the distro here? I think it might be a good idea if we can say that the container runs on Ubuntu X.X LTS or Redhat X.X.

Reply:
I took a shot at fixing this, but not sure I love it.